gallery_widgets/lib/features/text_field/widgets/text_field_2_views.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gallery_widgets/features/text_field/cubit/text_field_cubit.dart';
+
+class TextField2Views extends StatelessWidget {
+  const TextField2Views({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        vertical: 20,
+        horizontal: 10,
+      ),
+      child: BlocBuilder<TextFieldCubit, TextFieldState>(
+        builder: (context, state) {
+          return Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              buildTextField(state: state),
+              const SizedBox(height: 20),
+              buildTextField(hasBorder: true, state: state),
+            ],
+          );
+        },
+      ),
+    );
+  }
+
+  Widget buildTextField(
+      {bool hasBorder = false, required TextFieldState state}) {
```

Comment:
You should not use the state like this => use BlocBuilder for every widget using the state